web-setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
+
+# Author : Anthony Bartman
+# Description : This will install all neccessary files for
+# our Flask backend server. Mostly from this tutorial: 
+# https://www.codementor.io/@dongido/how-to-build-restful-apis-with-python-and-flask-12qto530jd
+
+echo ""
+echo ""
+echo "Running Flask Backend Install script."
+echo "This may take a few moments..."
+echo ""
+
+#Use Python 3.7 on board, not using Python2
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 7
+
+#Install Pip (If Necessary)
+sudo apt update
+sudo apt install python3-pip
+pip3 --version
+echo " - Installed pip3"
+
+#Install Virtual Env. to create isolated Python Env.
+#WE DO NOT NEED A VIRTUAL ENVIRONMENT FOR OUR SENIOR DESIGN PROGRESS
+# pip3 install virtualenv
+# virtualenv --version
+# echo " - Installed virtualenv"
+# Create Virtual Environment
+# python -m venv env
+# echo " - Virtual Environment Created"
+
+#Create Web Directory w/ base files(Front and Back end)
+mkdir web && cd web && mkdir backend && mkdir frontend && cd backend && mkdir resources && mkdir templates
+touch start.sh app.py run.py config.py requirements.txt && cd resources && touch Hello.py && cd ../ && cd templates && touch hello.html && cd ../
+echo " - Web Directory and base files created"
+
+#Installing other dependencies for Flask RESTful API
+echo "flask==0.12.2 
+flask_restful==0.3.6 
+flask_script==2.0.6
+marshmallow==2.14.0
+flask_marshmallow==0.8.0" > requirements.txt
+pip3 install -r requirements.txt
+echo " - Dependencies Installed Globally"
+
+#Setting up server files
+echo "echo \" || STARTING SERVER ||\"
```

Comment:
init should just setup the server stuff and probably not start it right away. starting should get taken care of by `files/init/start.sh` Unless the first time you start the server it's different from every additional time?